create_pdfs.py

```diff
@@ -97,31 +122,39 @@ def main():
             name_list = NAMES
 
         for name in name_list:
-            cert = CertificateGen(
-                course,
-                args.template_file,
-                aws_id=settings.CERT_AWS_ID,
-                aws_key=settings.CERT_AWS_KEY,
-                dir_prefix=pdf_dir,
-                long_org=args.long_org,
-                long_course=args.long_course,
-                issued_date=args.issued_date,
-            )
-            title = args.assign_title or None
+            if not CertificateGen.reuse_cert(cert, course, designation):
```

Comment:
This was specifically added here because it exists in certificate_agent.  Not for optimization.   This keeps the development cert creation closer to what happens in production.